guide/index.md

```diff
@@ -94,7 +87,7 @@ export default defineConfig({
 ```
 :::
 
-If you want to check it in `dev`, add the `devOptions` option to the plugin configuration (you will have the [Web App Manifest](https://developer.mozilla.org/en-US/docs/Web/Manifest) and the generated service worker):
+If you want to check it in `dev`, add the `devOptions` option to the plugin configuration (you will have the [Web App Manifest][webmanifest] and the generated service worker):
```

Comment:
add the msdn link back: md files will be transpiled to html on build time

Reply:
Not sure I understand. If the md is converted to HTML, doesn't it automatically replace the link references with the original link? If so, why not have cleaner markup?